apps/condo/domains/acquiring/constants/constants.js

```diff
@@ -0,0 +1,7 @@
+const RESIDENT_SUGGEST_SERVICE_PROVIDER_WINDOW_IN_SEC = 60 * 60 // 1 hour in seconds
```

Comment:
You need to use your own constants, like
RESIDENT_CALCULATE_FEE_FOR_RECEIPT_WINDOW_IN_SEC